src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/ProjectSystem/ProjectSnapshotManagerBenchmarkBase.cs

```diff
@@ -93,9 +93,11 @@ private static IReadOnlyList<TagHelperDescriptor> ReadTagHelpers(string filePath
 
         private class TestForegroundDispatcher : ForegroundDispatcher
         {
-            public override bool IsForegroundThread => true;
+            public override bool IsSpecializedForegroundThread => true;
```

Comment:
I renamed this from `IsForegroundThread` -> `IsSpecializedForegroundThread` to better reflect that we're referring to a specialized thread here as opposed to the UI thread.